src/bin/commands/new.js

```diff
@@ -14,17 +14,17 @@ const inquirer = require('inquirer');
 const STABLE_VERSIONS = {
```

Comment:
Did you validate these changes are stable?

Reply:
Running few more validations, will confirm when everything works perfect. please wait for ack.

Reply:
downgraded `standard-version` for now. every other update here works correctly.

Reply:
Published `create-react-styleguide@prerelease` for final testing

Reply:
nitpick: I'd squash the "revert standard-version to previous release" commit into your first commit just for a clean commit history